code/gamemode/Game.cs

```diff
@@ -26,13 +26,17 @@ public partial class Game : Sandbox.Game
 
         public TTTMapSettings MapSettings { get; private set; }
 
-        [ConVar.Replicated]
+        [ConVar.Replicated("ttt_debug")]
         public bool Debug { get; set; } = false;
 
         public Game()
         {
             Instance = this;
 
+            Utils.RNG.Next();
+            Utils.RNG.Next();
+            Utils.RNG.Next();
```

Comment:
What is this for?

Reply:
Warmup of the RNG, if it's not needed in C#, feel free to remove :)

Reply:
nah shouldn't be needed, never actually seen something like this before. Was it done in .lua before?

Reply:
yea :D